regulations/management/commands/setup_cors.py

```diff
@@ -17,7 +17,7 @@ def handle(self, *args, **options):
             CORSConfiguration={
                 'CORSRules': [
                     {
-                        'AllowedMethods': ['PUT'],
+                        'AllowedMethods': ['GET', 'PUT'],
```

Comment:
On second thought, the permissions granted below make a lot more sense than firewalling by bucket. Nevermind!